### PR TITLE
Bulgarian translation part 1

### PR DESCRIPTION
**Pull request by xpucuto**
I have translated till  You secret API, i will do my best to finish the rest in the next few days. As i see the translation will be int he new version of Nightspot where yo can edit you Events.  
Please let me know if have to change anything in the language in order to be viewed correctly . 
Hristina

### CODE REVIEW

File: lib/language.js [hunk 1]
```diff
@@ -422,6 +473,7 @@ function init() {
       ,fr: 'Percentiles'
       ,pt: 'Percentis'
       ,ro: 'Grafic percentile'
+      ,бг: "Процентна графика"
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/819607010/issues/source?bid=2452379&fileBranchId=2748601#l476)

Comment (xpucuto) [inline lib/language.js]: Oh sorry I must have done it by mistake . 
I hope there not many like that . I will also check it . 
Hristina

**Comment by jasoncalabrese** [comment]
@xpucuto, looks like you used a mix of `бг` and `bg` for the language code.  I think we should stick to ascii for the language code to prevent any issues with env vars in different environments.

Also take a look at the quote issue above, just need to change the quotes on that 1 line I think.

**Comment by xpucuto** [comment]
Ok i will fix that and send a new request . Thank you

**Comment by jasoncalabrese** [comment]
This was replaced with #759

**Pull request closed without merging**